core/templates/app.constants.ts

```diff
@@ -168,7 +168,7 @@ export class AppConstants {
     QUESTION: 'question'
   };
 
-  public static MISCONCEPTION_NAME_CHAR_LIMIT = 100;
+  public static MISCONCEPTION_NAME_CHAR_LIMIT = 50;
```

Comment:
is this defined somewhere i the backend as well, if yes then I would suggest using constants.ts

Reply:
No, in the backend it uses the general `utils.require_valid_name` function to validate which limits it to 50 characters.

Reply:
Then I would add a new constant that would be used both in utils and in here. Something like `GENERIC_NAME_FIELD_CHAR_LIMIT`.

Reply:
@aks681 Could you please respond here?

Reply:
Yeah, will add the constant.

Reply:
Done

Reply:
@vojtechjelinek Removed `GENERIC_NAME_FIELD_CHAR_LIMIT` as well and used a new constant `MAX_CHARS_IN_MISCONCEPTION_NAME` since the limit had to be increased to 100 according to #9234.